File: SRPTests/TestScripts/SimplePixelShader.py
# Test very simple pixel shader.

vs = ri.LoadShader("ConstantColour.hlsl", "VS", "vs_4_0")
ps = ri.LoadShader("ConstantColour.hlsl", "PS", "ps_4_0")

def RenderFrame(context):
	context.DrawFullscreenQuad(vs, ps)

ri.SetFrameCallback(RenderFrame)

File: SRPTests/TestScripts/CS/SimplePixelShader.cs
// Test very simple pixel shader.

var vs = ri.CompileShader("ConstantColour.hlsl", "VS", "vs_4_0");
var ps = ri.CompileShader("ConstantColour.hlsl", "PS", "ps_4_0");

ri.SetFrameCallback(context =>
{
	context.DrawFullscreenQuad(vs, ps);
});
